### PR TITLE
Implemented long tap/press event listener / handler. Going to self location (fab) will zoom in to 1st level resolution.

### PR DESCRIPTION
**Pull request by BarAgam**
Implemented long tap/press event listener / handler.
Going to self location (fab) will zoom in to 1st level resolution.

### CODE REVIEW

File: app/src/main/java/com/teamagam/gimelgimel/app/map/esri/EsriGGMapView.java [hunk 1]
```diff
@@ -219,4 +226,27 @@ private boolean isEntityClicked(int graphicId) {
...
+        @Override
+        public boolean onLongPress(float screenX, float screenY) {
+            PointGeometry pointGeometry = getClickedPointGeometry(screenX, screenY);
+            notifyOnLocationChosen(pointGeometry);
+            return false;
+        }
+
+        private PointGeometry getClickedPointGeometry(float screenX, float screenY) {
+            Point point = EsriGGMapView.this.toMapPoint(screenX, screenY);
+            Point wgs84Point = (Point) GeometryEngine.project(point, getSpatialReference(),
```
Comment (teamagam5) [inline app/src/main/java/com/teamagam/gimelgimel/app/map/esri/EsriGGMapView.java]: Extract projectToWGS84(Geometry p)

Comment (teamagam5) [reply app/src/main/java/com/teamagam/gimelgimel/app/map/esri/EsriGGMapView.java]: Then rename wgs84Point to p?

File: app/src/debug/java/com/teamagam/gimelgimel/app/common/utils/Constants.java [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 
 public class Constants {
-    public static final float LOCATE_ME_BUTTON_ALTITUDE_METERS = 500;
+    public static final float LOCATE_ME_BUTTON_RESOLUTION_LEVEL = 1;
```
Comment (teamagam5) [inline app/src/debug/java/com/teamagam/gimelgimel/app/common/utils/Constants.java:5]: I c u made this change in the release too, is it needed here?

**Review by teamagam5 — APPROVED** [review]

File: app/src/debug/java/com/teamagam/gimelgimel/app/common/utils/Constants.java (hunk 2 above)
Comment (BarAgam) [reply app/src/debug/java/com/teamagam/gimelgimel/app/common/utils/Constants.java:5]: Yes.
Circle CI builds the application in release mode. If it wont find his constant it'll fail.

File: app/src/main/java/com/teamagam/gimelgimel/app/map/esri/EsriGGMapView.java (hunk 1 above)
Comment (BarAgam) [reply app/src/main/java/com/teamagam/gimelgimel/app/map/esri/EsriGGMapView.java]: I'll extract.
But why the renaming ?

**Pull request merged**